app.py

```diff
@@ -190,3 +194,25 @@ def list_repo():
             "text": message,
         }
     )
+
+@app.route("/projetcs", methods=["POST"])
+def list_project():
+    data = request.form
+    text = data.get("text")
+    user_name = data.get("user_name")
+    project_name = text.strip().lower()
+
+    project = project_data.get(project_name)
+
+    if project:
+        project_list = "\n".join(project)
+        message = f"Hello {user_name}, here the information about '{project_name}':\n{project_list}"
+    else:
+        message = f"Hello {user_name}, the project '{project_name}' is not recognized. Please try again."
```

Comment:
```suggestion
        message = f"Hello {user_name}, the project '{project_name}' is not recognized. Please try different query."
```
or similar wording instead of "try again"